terraform/shared/modules/egress-proxy/prepare-proxy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# Exit if any step fails
+set -e
+
+popdir=$(pwd)
+
+# Portable construct so this will work everywhere
+# https://unix.stackexchange.com/a/84980
+tmpdir=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
+cd "$tmpdir"
+
+# Grab a copy of the main branch
+curl -s -L https://github.com/GSA-TTS/cg-egress-proxy/archive/refs/heads/main.zip --output main.zip 
```

Comment:
We should be grabbing a particular release tag (which implies that the repository is making tags/releases).